### PR TITLE
Refactor/renamed method name to follow ruby conventions

### PR DESCRIPTION
**Pull request by sebastiandeutsch**
I've done a couple of small refactorings so that the code gets more Ruby idiomatic.

### CODE REVIEW

File: spec/support/load_and_extend_graph_from_file.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 def load_and_extend_graph_from_file(filename)
   session = Tensorflow::Session.new()
   graph = Tensorflow::Graph.new()
-  graph.read(File.dirname(__FILE__)+'/../example_graphs/'+filename)
+  graph.read(File.join(File.dirname(__FILE__), '../', 'example_graphs', filename))
```
Comment (nethsix) [inline spec/support/load_and_extend_graph_from_file.rb]: Removing '/' from the '../' seems to work as well. Is using '../' a Ruby coding best practice? Please advice.

File: spec/support/load_and_extend_graph_from_file.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+def load_and_extend_graph_from_file(filename)
```
Comment (nethsix) [inline spec/support/load_and_extend_graph_from_file.rb]: Would it be better to define `load_and_extend_graph_from_file` in a namespace using a `Module`? Thoughts anyone?

**Comment by nethsix** [comment]
@sebastiandeutsch @Arafatk Reviewed and left some comments about things that need your help.

File: spec/support/load_and_extend_graph_from_file.rb (hunk 1 above)
Comment (zverok) [inline spec/support/load_and_extend_graph_from_file.rb]: Yep, I think `/` is clearly redundant.

**Comment by Arafatk** [comment]
@nethsix @zverok @sebastiandeutsch Thanks

File: spec/support/load_and_extend_graph_from_file.rb (hunk 1 above)
Comment (sebastiandeutsch) [inline spec/support/load_and_extend_graph_from_file.rb]: Cool. Makes sense.

File: spec/support/load_and_extend_graph_from_file.rb (hunk 2 above)
Comment (sebastiandeutsch) [inline spec/support/load_and_extend_graph_from_file.rb]: I'm usually starting with this and create modules when things get messy.

http://ryanbigg.com/2013/02/about-spec-support/

**Pull request merged**